### PR TITLE
docs: update Playwright automated-testing guide

### PR DESCRIPTION
**Pull request by mxschmitt**
#### Description of Change

This PR refreshes the Playwright automated-testing guide for 2024. Relevant comments are attached on the files tab.

cc @codebytere

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] relevant documentation, tutorials, templates and examples are changed or added

#### Release Notes

Notes: none

### CODE REVIEW

File: docs/tutorial/automated-testing.md [hunk 1]
```diff
@@ -196,41 +196,27 @@ support via Electron's support for the [Chrome DevTools Protocol][] (CDP).
 
 ### Install dependencies
 
-You can install Playwright through your preferred Node.js package manager. The Playwright team
-recommends using the `PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD` environment variable to avoid
-unnecessary browser downloads when testing an Electron app.
-
-```sh npm2yarn
-PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 npm install --save-dev playwright
```
Comment (mxschmitt) [inline docs/tutorial/automated-testing.md:204]: We don't install the browsers automatically anymore, so the environment variable is not required.

File: docs/tutorial/automated-testing.md [hunk 2]
```diff
@@ -196,41 +196,27 @@ support via Electron's support for the [Chrome DevTools Protocol][] (CDP).
...
 ```
 
 :::caution Dependencies
-This tutorial was written `playwright@1.16.3` and `@playwright/test@1.16.3`. Check out
+This tutorial was written with `@playwright/test@1.41.1`. Check out
 [Playwright's releases][playwright-releases] page to learn about
 changes that might affect the code below.
 :::
 
-:::info Using third-party test runners
```
Comment (mxschmitt) [inline docs/tutorial/automated-testing.md:220]: We discourage other test-runners and removed the guide a couple of releases ago.

File: docs/tutorial/automated-testing.md [hunk 3]
```diff
@@ -196,41 +196,27 @@ support via Electron's support for the [Chrome DevTools Protocol][] (CDP).
...
-:::
-
 ### Write your tests
 
 Playwright launches your app in development mode through the `_electron.launch` API.
 To point this API to your Electron app, you can pass the path to your main process
 entry point (here, it is `main.js`).
 
 ```js {5} @ts-nocheck
-const { _electron: electron } = require('playwright')
```
Comment (mxschmitt) [inline docs/tutorial/automated-testing.md:232]: `@playwright/test` re-exports everything inside the `playwright` package.

**Review by ckerr — APPROVED** [review]
Thanks!

**Pull request merged**